aw_core/log.py

```diff
@@ -32,6 +32,11 @@ def setup_logging(
     root_logger.setLevel(logging.DEBUG if verbose else logging.INFO)
     root_logger.handlers = []
 
+    # run with LOG_LEVEL=DEBUG to customize log level across all AW components
+    log_level = os.environ.get('LOG_LEVEL')
+    if log_level:
+        root_logger.setLevel(logging.__getattribute__(log_level.upper()))
```

Comment:
```suggestion
        if hasattr(log_level, log_level.upper()):
            root_logger.setLevel(getattr(logging, log_level.upper()))
        else:
            root_logger.warning(f'No logging level called {log_level} (as specified in env var)')
```

Reply:
Ah, wasn't aware of those functions! Thanks.